samplePDF/samplePDFFDBase.h

```diff
@@ -52,27 +50,14 @@ class samplePDFFDBase :  public samplePDFBase
 
   // Setup and config functions
   void UseNonDoubledAngles(bool ans) {doubled_angle = ans;};
```

Comment:
isn't this a property of the oscillation calculator, rather than the samplePDF?

Reply:
this has been removed :)